src/rpc/blockchain.cpp

```diff
@@ -448,14 +360,14 @@ UniValue getblockdeltas(const UniValue& params, bool fHelp)
             "      ]\n"
             "    }, ...\n"
             "  ],\n"
-            "  \"time\": n,\n"
-            "  \"mediantime\": n,\n"
-            "  \"nonce\": \"hexstring\",\n"
-            "  \"bits\": \"hexstring\",\n"
-            "  \"difficulty\": ,\n"
-            "  \"chainwork\": \"hexstring\",\n"
-            "  \"previousblockhash\": \"hash\",\n"
-            "  \"nextblockhash\": \"hash\"\n"
+            "  \"time\" : n,                  (numeric) The block version\n"
+            "  \"mediantime\": n,             (numeric) The most recent blocks' ave time\n"
+            "  \"nonce\" : \"nonce\",           (hex string) The nonce\n"
+            "  \"bits\" : \"1d00ffff\",         (hex string) The bits\n"
+            "  \"difficulty\": n,             (numeric) the current difficulty\n"
+            "  \"chainwork\": \"xxxx\"          (hex string) total amount of work in active chain\n"
+            "  \"previousblockhash\" : \"hash\",(hex string) The hash of the previous block\n"
+            "  \"nextblockhash\" : \"hash\"     (hex string) The hash of the next block\n"
```

Comment:
Given how many of these cleanup changes there are, they should all be in a separate commit from the addition of `getblockhashes`.